DiscoverySampleApp/DiscoverySampleApp/Players/PlayerViewController.m

```diff
@@ -0,0 +1,140 @@
+//
+//  MasterViewController.m
+//  OoyalaSkin
+//
+//  Created on 6/3/15.
+//  Copyright © 2015 Ooyala, Inc. All rights reserved.
+//
+
+#import "PlayerViewController.h"
+
+#import <OoyalaSkinSDK/OoyalaSkinSDK.h>
+#import <OoyalaSDK/OoyalaSDK.h>
+#import "DemoSettings.h"
+#import "PlayerSelectionOption.h"
+
+@interface PlayerViewController ()
+
+@property (nonatomic) IBOutlet UIView *videoView;
+@property (nonatomic) IBOutlet UILabel *textLabel;
+
+@property (nonatomic) OOSkinViewController *skinController;
+@property (nonatomic) OOOoyalaPlayer *ooyalaPlayer;
+@property (nonatomic) NSString *embedCode;
+@property (nonatomic) NSString *nib;
+@property (nonatomic) NSString *pcode;
+@property (nonatomic) NSString *playerDomain;
+@property (nonatomic) DemoSettings *configuration;
+
+@end
+
+@implementation PlayerViewController
+
+- (void)loadView {
+  [super loadView];
+  
+  self.nib = @"OOplayer";
+  self.configuration = [DemoSettings new];
+  self.pcode         = self.configuration.playerParameters[@"pcode"];
+  self.playerDomain  = self.configuration.playerParameters[@"domain"];
+  self.embedCode     = self.configuration.initasset[@"embedCode"];
+  [NSBundle.mainBundle loadNibNamed:self.nib owner:self options:nil];
+}
+
+- (void)setCustomSkin {
+  // Setting Discovery on player
+  OODiscoveryOptions *discoveryOptions = [[OODiscoveryOptions alloc] initWithType:OODiscoveryTypePopular
+                                                                            limit:10
+                                                                          timeout:60];
+  // Modifying PlayerViewController to use Ooyala Skin
+  NSURL *jsCodeLocation = [NSBundle.mainBundle URLForResource:@"main" withExtension:@"jsbundle"];
+//  NSURL *jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle?platform=ios&dev=true"];
```

Comment:
same

Reply:
It's an option for `main.jsbundle` to switch between a local file and a running on localhost